### DISCUSSION TITLE
What do you mean by Solidity being "a costly updating language"?

### DISCUSSION THREAD

**Discussion by marshallthomson** [smartcontractkit/full-blockchain-solidity-course-py#673] [Q&A]
I wasn't aware there was a cost to update languages. 

Or do you mean there are syntax changes that would cost time/money to update? So being able to compile at a previous version can be advantageous instead of refactoring?

Or, are you referring to the cost of updating on Ethereum?

Sorry, complete n00b here...

**Comment by cromewar** [comment]
Hello @marshallthomson sorry for the delay, this is actually a tricky question.

Every language is costly if you look at how many developers are working on it and the cost you have for their salaries and so on. so I don't think you should take this as literally.
Is also true that every transaction cost money but is more related to the blockchain rather than the language, so could you please give us a hand on this one @PatrickAlphaC ?

**Reply by PatrickAlphaC** [reply]
Great question. What is the context of this? I could answer it at face value... but I'm curious where I said it if it's related to the course...
